web-api/src/email/handleBounceNotificationsLambda.js

```diff
@@ -0,0 +1,24 @@
+const createApplicationContext = require('../applicationContext');
+
+/**
+ * This lambda handles SNS notifications that occur whenever a service Email bounces. We
+ * may need to take action when these events happen.
+ *
+ * @param {object} event the AWS event object received that includes any messages from our SNS subscription
+ * @returns {Promise<*>|undefined} the response to the topic
+ */
+exports.handleBounceNotificationsLambda = async event => {
+  const applicationContext = createApplicationContext({});
+
+  const records = event.Records.map(record => ({
+    ...JSON.parse(record.Sns.Message),
+  }));
+
+  return await Promise.all(
```

Comment:
This is a great question, and now I fear lost because I have merged in the latest from `staging` and this PR is polluted. I want to figure out a good answer to this question because my instinct is that `genericHandler` isn't the right wrapper for this; I don't believe one currently exists. That handler is for the API, whereas this is different lambda. I opted against using one for the Seal Case in Lower Environment lambda. 

Going to give this some 🤔

Reply:
Ok that makes sense we would want to keep consistent with other related lambdas!